ethyca/fides
# 2129-Side nav disappears when viewing request details

Closes #2129 

### Code Changes

- Executed npm audit fix
- Created /privacy-requests/[id] route
- Ensured side nav is displayed when viewing privacy request details
- Refactored `subject-request` folder files to `privacy-requests` folder now

### Steps to Confirm

1. Execute `nox -s test_env`
2. Open browser tab to http://localhost:3000 (Fides)
3. Login with **fidestest** account
4. Click the **Review privacy requests** card
5. Open another browser tab to http://localhost:3001 (Privacy Center)
6. Click **Access your data** card
7. Enter **Name**, **Email**, and **Phone** values and click the **Continue** button
8. On the Fides tab, refresh your page. New DSR record should be displayed.
9. Click the **View Details** menu option for the given DSR record
10. Verify the **Privacy requests** side nav is displayed to the left of the DSR record details

### Pre-Merge Checklist

* Documentation:

### Screen Shot
![Screen Shot 2023-01-06 at 11 56 36 AM](https://user-images.githubusercontent.com/68459950/211060017-d7154c0d-e586-4b36-8750-299dae05e245.png)
